scripts/coverage.js

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env node
 
+const { execSync } = require('child_process');
 const { runCoverage } = require('@openzeppelin/test-environment');
 
-runCoverage(['mocks'], 'npm run compile', './node_modules/.bin/mocha --exit --timeout 10000 --recursive');
+runCoverage(['mocks'], 'npm run compile', './node_modules/.bin/mocha --exit --timeout 10000 --recursive'.split(' '));
+
+if (process.env.CI) {
+  execSync('curl -s https://codecov.io/bash | bash -s -- -C "$CIRCLE_SHA1"');
```

Comment:
This is so we see the logs in CI.

```suggestion
  execSync('curl -s https://codecov.io/bash | bash -s -- -C "$CIRCLE_SHA1"', { stdio: 'inherit' });
```